tests/migration_test.go

```diff
@@ -336,7 +336,87 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 		}
 	}
 
+	setMigrationBandwidthLimitation := func(migrationBandwidth resource.Quantity) {
+		cfg := getCurrentKv()
+		cfg.MigrationConfiguration.BandwidthPerMigration = &migrationBandwidth
+		tests.UpdateKubeVirtConfigValueAndWait(cfg)
+	}
+
 	Describe("Starting a VirtualMachineInstance ", func() {
+
+		var pvName string
+		BeforeEach(func() {
+			pvName = "test-nfs" + rand.String(48)
+		})
+
+		const guestAgentMigrationTestName = " should be migrated successfully, using guest agent on VM "
+		guestAgentMigrationTestFunc := func(mode v1.MigrationMode) {
+			memoryRequestSize := resource.MustParse(fedoraVMSize)
+			if mode == v1.MigrationPostCopy {
```

Comment:
Now you can extract this config update to `BeforeEach` block of "migration postcopy". It would speed up the test and make it less likely to flake(propagating the config use to flake)

Reply:
Good idea, thanks!